src/server/pfs/s3/driver.go

```diff
@@ -103,10 +75,12 @@ func (d *MasterDriver) listBuckets(pc *client.APIClient, r *http.Request, bucket
 	return nil
 }
 
-func bucketNameToCommit(bucketName string) *pfs.Commit {
-	var id string
-	branch := "master"
-	var repo *pfs.Repo
+func bucketNameToCommit(bucketName string) (*pfs.Commit, error) {
+	var (
+		id     string
+		branch string
+		repo   *pfs.Repo
+	)
 
 	// the name is [commitID.][branch. | branch.type.]repoName
```

Comment:
I believe this comment is now out of date

Reply:
Fixed, thanks